src/Features/CSharp/Portable/ReplacePropertyWithMethods/CSharpReplacePropertyWithMethodsService.cs

```diff
@@ -132,8 +132,13 @@ public override SyntaxNode GetPropertyDeclaration(SyntaxToken token)
 
             if (propertyDeclaration.ExpressionBody != null)
             {
-                var returnStatement = SyntaxFactory.ReturnStatement(propertyDeclaration.ExpressionBody.Expression)
-                                                   .WithSemicolonToken(propertyDeclaration.SemicolonToken);
+                var returnKeyword = SyntaxFactory.Token(SyntaxKind.ReturnKeyword)
+                                                 .WithTrailingTrivia(propertyDeclaration.ExpressionBody.ArrowToken.TrailingTrivia);
+
+                var returnStatement = SyntaxFactory.ReturnStatement(
+                    returnKeyword, 
+                    propertyDeclaration.ExpressionBody.Expression,
+                    propertyDeclaration.SemicolonToken);
```

Comment:
The trivia from the expression is maintained because we pass it along unchanged: 

```SyntaxFactory.ReturnStatement(
+                    returnKeyword, 
+                    propertyDeclaration.ExpressionBody.Expression,
```